Library/ViewModels/VideoViewModelTests.swift

```diff
@@ -226,6 +226,18 @@ internal final class VideoViewModelTests: TestCase {
 
     self.vm.inputs.rateChanged(toNew: self.playRate, atTime: self.halfwayTime)
 
+    XCTAssertEqual([
+      "Video Playback Started",
+      "Video Playback Started"
```

Comment:
Why are there two events here if the playButton input was only selected once?

Reply:
Also noticing when I test this on the debugger that occasionally two events are being fired at once. Wondering if this is related. Can you test this out a bit more and let me know what you find?